### DISCUSSION TITLE
[Feature] Search for more than one person on a photo

### DISCUSSION THREAD

**Discussion by fwmone** [immich-app/immich#7419] [Feature Request]
### The feature

Thanks for the new search interface in release 1.95, it's a great addition!

But, when selecting multiple persons, they are OR'd instead of AND'd. Hence, you select persons A and B and get photos with A, B or both. It would be great to have the option to AND them alternatively: Select persons A and B only shows photos that contain BOTH persons. At least for me, this is the more natural use of the person search (e. g. "get photos of my wife, my son and me"). 

Thanks!

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7306
